lib/git.zsh

```diff
@@ -70,6 +70,13 @@ function git_commits_ahead() {
   fi
 }
 
+# Gets the number of commits behind remote
+function git_commits_behind() {
+  if git rev-parse --git-dir > /dev/null 2>&1; then
```

Comment:
This function should use original git command.
plz change to this:
before:
```
  if git ...
```

after:
```
  if command git ...
```